.cirrus.yml

```diff
@@ -989,6 +989,61 @@ test_image_build_task:
     env:
         DRYRUN: 1
 
+fcos_image_build_task: &fcos_image-build
+    name: "Build FCOS image $CTXDIR"
+    alias: fcos_image_build
+    # Some of these container images take > 1h to build, limit
+    # this task to a specific Cirrus-Cron entry with this name.
+    # Docs: ./contrib/cirrus/CIModes.md
+    only_if: $CIRRUS_CRON == ''
```

Comment:
This task is meant to run before merge. The `fcos_image_build_push_task` is meant for after-merge.

Reply:
Hmm. I guess the `test_fcos_image_build_task` could be run before merge, `fcos_image_build_task` after merge and then the 3rd task I added with `push` in it could be just deleted, is that what you meant? But then I would want the `test_fcos_` task to always run.

Reply:
btw, if the `test_fcos_image_build_task` isn't automatically triggered, I guess there's no point to even having it in the list.